p2p-tests/contributor-fork-replication-1-test.ts

```diff
@@ -133,6 +133,14 @@ async function testcase(dataPath: string) {
     keyPassphrase: contributor.passphrase,
   });
 
+  // ALEX: We're not entirely sure when the announcement occurs. The daemon has
+  // a loop which announces every second, but we also need to allow some time
+  // for the announcement to be received by the seed and the replication to
+  // occur. In practice 5 seconds seems to work for me.
+  //
+  // I'm quite unhappy about this
+  await sleep(5000);
```

Comment:
The `withRetry` call below should be able to wait more efficiently. We continuously check until we’ve successfully replicated. If it times out we should just bump the time out in `withRetry`.

Reply:
Interestingly bumping the timeout in `withRetry` to 5000 doesn't work. This is strange as it seems like it should be semantically identical to this `sleep`.